statics/js/components/graph-layout.js

```diff
@@ -131,21 +116,20 @@ var LinkLabelLatency = Vue.extend({
         return;
       }
 
-      const limit = 10;
-      const context = Date.now() - maxClockSkewMillis;
-      const query = `G.Flows()` +
-        `.Context(${context})` +
-        `.Has("NodeTID", Within("${a.TID}", "${b.TID}"))` +
-        `.Has("RTT", NE(0))` + 
-        `.Limit(${limit})`;
-
-      this.$topologyQuery(query)
+      this.flowQuery(a.TID, undefined, 10)
         .then(function(flows) {
-          if (flows.length === 0) {
-            return;
-          }
-          flows = flows.sort(self.flowCompare);
-          self.updateLatency(link, flows);
+          for (let i in flows) {
```

Comment:
@hunchback I would rather do for the first request Flows().Has('NodeTID', A).Sort().Limit(10), so not using context. It will return the most recent flows.

Reply:
@safchain - ok I have replaced Context() with Sort()